Dockerfile

```diff
@@ -38,44 +38,46 @@ ENV BIOMAJ_CONFIG=/etc/biomaj/config.yml
 
 RUN mkdir -p /var/log/biomaj
 
-RUN pip3 install --no-cache-dir setuptools --upgrade && \
+RUN pip3 install --no-cache-dir pip --upgrade && \
+    pip3 install --no-cache-dir setuptools --upgrade && \
     pip3 install --no-cache-dir greenlet==0.4.17 && \
-    pip3 install --no-cache-dir gevent==1.4.0 && \
+    # pip3 install --no-cache-dir gevent==1.4.0 && \
     pip3 install --no-cache-dir graypy && \
     pip3 install --no-cache-dir pymongo==3.12.3 && \
     pip3 install --no-cache-dir redis==3.5.3 && \
     pip3 install --no-cache-dir wheel && \
     pip3 install --no-cache-dir PyYAML==5.4.1 && \
+    pip3 install --no-cache-dir protobuf==3.20.3 && \
     python3 -m pip install --no-cache-dir ftputil
 
 ENV SUDO_FORCE_REMOVE=yes
 RUN buildDeps="gcc python3-dev protobuf-compiler" \
     && set -x \
     && apt-get update \
     && apt-get install -y $buildDeps --no-install-recommends \
-    && cd /root/biomaj-core && python3 setup.py build && pip3 install --no-cache-dir . \
```

Comment:
Now you can remove the git clone commands at lines 24~35

Reply:
I could but then @mboudet's `cd /root/biomaj-process/biomaj_process/message && protoc --python_out=. procmessage.proto` causes an error during the `docker compose build`.

Reply:
Either we build it here, or we build it in the biomaj_process and biomaj_download repo (like my branch)
Might be complicated to build it in each repo (probably the build environnement would need to match the dockerfile, but that means these two repo will only work with docker)